tests/test_config.py

```diff
@@ -126,19 +126,21 @@ def test_asgi_version(app, expected_interface):
     [
         pytest.param(None, None, id="use_colors_not_provided"),
         pytest.param(True, True, id="use_colors_enabled"),
+        pytest.param("invalid", None, id="use_colors_invalid_value"),
```

Comment:
Swapped the test cases, as it breaks the suite in `master`

More: [comment link](https://github.com/encode/uvicorn/pull/814#issuecomment-709948830)